mmv1/products/bigquerydatapolicy/api.yaml

```diff
@@ -0,0 +1,96 @@
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://bigquerydatapolicy.googleapis.com/v1beta1/
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: BigQuery Data Policy API
+    url: https://console.cloud.google.com/apis/library/bigquerydatapolicy.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: "DataPolicy"
+    min_version: beta
+    base_url: projects/{{project}}/locations/{{location}}/dataPolicies
+    create_url: projects/{{project}}/locations/{{location}}/dataPolicies
+    self_link: projects/{{project}}/locations/{{location}}/dataPolicies/{{data_policy_id}}
+    update_verb: :PATCH
+    update_mask: true
+    description: A BigQuery Data Policy
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "Official Documentation": "https://cloud.google.com/bigquery/docs/column-data-masking-intro"
+      api: "https://cloud.google.com/bigquery/docs/reference/bigquerydatapolicy/rest/v1beta1/projects.locations.dataPolicies/create"
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      exclude: false
+      method_name_separator: ":"
+      fetch_iam_policy_verb: :POST
+      parent_resource_attribute: "data_policy_id"
+      import_format:
+        [
+          "projects/{{project}}/locations/{{location}}/dataPolicies/{{data_policy_id}}",
+          "{{data_policy_id}}",
+        ]
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |-
+          Resource name of this data policy, in the format of projects/{project_number}/locations/{locationId}/dataPolicies/{dataPolicyId}.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: dataPolicyId
+        description: |-
+          User-assigned (human readable) ID of the data policy that needs to be unique within a project. Used as {dataPolicyId} in part of the resource name.
+        required: true
+        input: true
+      - !ruby/object:Api::Type::String
+        name: location
+        description: |
+          The name of the location of the data policy.
+        required: true
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: policyTag
+        description: |-
+          Policy tag resource name, in the format of projects/{project_number}/locations/{locationId}/taxonomies/{taxonomyId}/policyTags/{policyTag_id}.
+        required: true
+      - !ruby/object:Api::Type::Enum
+        name: dataPolicyType
+        description: |
+          The enrollment level of the service.
+        required: true
+        values:
+          - :COLUMN_LEVEL_SECURITY_POLICY
+          - :DATA_MASKING_POLICY
+      - !ruby/object:Api::Type::NestedObject
+        name: "dataMaskingPolicy"
+        description: |
+          The data masking policy that specifies the data masking rule to use.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: "predefinedExpression"
```

Comment:
This should be required as it is the only sub-field of dataMaskingPolicy

Reply:
added